### PR TITLE
[stable14] use a dummy database name for the default postgres database name

### PR DESCRIPTION
**Pull request by rullzer**
Backport of #11262

For some reason the docker image does not setup the permissions correctly,
by using a different name the nextcloud installer will create the database instead
with the correct permissions

Signed-off-by: Robin Appelman <robin@icewind.nl>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
:elephant: :+1:

File: .drone.yml [hunk 1]
```diff
@@ -284,7 +284,7 @@ pipeline:
     image: nextcloudci/php7.1:php7.1-16
     commands:
       - sleep 10 # gives the database enough time to initialize
-      - NOCOVERAGE=true TEST_SELECTION=DB ./autotest.sh pgsql
+      - POSTGRES=${POSTGRES} NOCOVERAGE=true TEST_SELECTION=DB ./autotest.sh pgsql
```
Comment (danxuliu) [inline .drone.yml]: Why is this needed? And is it not needed in https://github.com/nextcloud/server/pull/11421/files#diff-3216dfff0ed3e301453e6799e8c367e2R278 too?

Comment (rullzer) [reply .drone.yml]: I just cherry picked...

Comment (rullzer) [reply .drone.yml]: @icewind1991 any idea?

**Review by MorrisJobke — APPROVED** [review]
